i18n/de.yaml

```diff
@@ -0,0 +1,636 @@
+# Language names TODO: find a better way to do this.
+- id: en
+  translation: "English"
+- id: fr
+  translation: "Français"
+- id: ru
+  translation: "Русский"
+- id: zh-hans
+  translation: "中文"
+- id: zh-hant
+  translation: "繁體中文"
+- id: de
+  translation: "Deutsch"
+
```

Comment:
Could you add the translation for `de` to `en.yaml`, and then delete this line and above in this file?

It's an oddity with the way the translation library works with Hugo. Not entirely sure that it's not a bug to be honest, but nonetheless the workaround is easy.